### PR TITLE
Add other languages in list

### PR DESCRIPTION
**Pull request by MatejMecka**
Added other languages that the commend referenced which was a Google Play support article

### CODE REVIEW

File: lib/models/language.dart [hunk 1]
```diff
@@ -36,11 +36,40 @@ abstract class GooglePlayLanguages {
   static const englishUnitedKingdom =
       Language('en-GB', 'English (United Kingdom)');
   static const englishUnitedStates =
-      Language('en-US', 'English (United States)');
+      Language('en-US', 'English (United State)');
```
Comment (Albert221) [inline lib/models/language.dart]: Regression here.

File: lib/models/language.dart [hunk 2]
```diff
@@ -41,6 +41,35 @@ abstract class GooglePlayLanguages {
   static const filipino = Language('fil', 'Filipino');
   static const finnish = Language('fi-FI', 'Finnish');
   static const french = Language('fr-FR', 'French');
+  static const french_canadian = Language('fr-CA', 'French (Canada)');
```
Comment (Albert221) [inline lib/models/language.dart]: Use camelCase

File: lib/models/language.dart [hunk 3]
```diff
@@ -41,6 +41,35 @@ abstract class GooglePlayLanguages {
...
+  static const hindi = Language('hi', 'Hindi');
+  static const icelandic = Language('is', 'Icelandic');
+  static const indonesian = Language('id', 'Indonesian');
+  static const italian = Language('it', 'Italian');
+  static const korean = Language('ko', 'Korean');
+  static const latvian = Language('lv', 'Latvian');
+  static const lithuanian = Language('lt', 'Lithuanian');
+  static const malay = Language('ms', 'Malay');
+  static const norwegian = Language('no', 'Norwegian');
+  static const portuguese_br = Language(' pt-BR', 'Portuguese (Brazil)');
```
Comment (Albert221) [inline lib/models/language.dart]: Leading space

File: lib/models/language.dart [hunk 4]
```diff
@@ -41,6 +41,35 @@ abstract class GooglePlayLanguages {
...
+  static const portuguese_pr = Language('pt-PT', 'Portuguese (Portugal) ');
```
Comment (Albert221) [inline lib/models/language.dart]: Trailing space

File: lib/models/language.dart (hunk 3 above)
Comment (Albert221) [reply lib/models/language.dart]: Also snake_case instead of camelCase

File: lib/models/language.dart (hunk 4 above)
Comment (Albert221) [reply lib/models/language.dart]: Also snake_case instead of camelCase

File: lib/models/language.dart [hunk 5]
```diff
@@ -72,5 +101,33 @@ abstract class GooglePlayLanguages {
...
+    slovak, 
+    slovenian,   
+    spanish_la,  
+    spanish,  
+    swahili,  
+    swedish,  
+    thai, 
+    ukrainian,  
+    vietnamese,  
+    zulu
```
Comment (Albert221) [inline lib/models/language.dart]: Add trailing comma

File: lib/models/language.dart [hunk 6]
```diff
@@ -41,6 +41,35 @@ abstract class GooglePlayLanguages {
...
+  static const malay = Language('ms', 'Malay');
+  static const norwegian = Language('no', 'Norwegian');
+  static const portuguese_br = Language(' pt-BR', 'Portuguese (Brazil)');
+  static const portuguese_pr = Language('pt-PT', 'Portuguese (Portugal) ');
+  static const romanian = Language('ro', 'Romanian');
+  static const russian = Language('ru', 'Russian');
+  static const serbian = Language('sr', 'Serbian');
+  static const slovak = Language('sk', 'Slovak');
+  static const slovenian = Language('sl', 'Slovenian'); 
+  static const spanish_la = Language('es-419', 'Spanish (Latin America)');
```
Comment (Albert221) [inline lib/models/language.dart]: Use camelCase instead of snake_case

File: lib/models/language.dart [hunk 7]
```diff
@@ -41,6 +41,35 @@ abstract class GooglePlayLanguages {
...
+  static const spanish = Language('es-ES', 'Spanish (Spain)');
+  static const swahili = Language('sw', 'Swahili');
+  static const swedish = Language('sv', 'Swedish');
+  static const thai = Language('th', 'Thai');
+  static const ukrainian = Language('uk', 'Ukrainian');
+  static const vietnamese = Language('vietnamese', 'Vietnamese');
+  static const zulu = Language('zu', 'Zulu');
+
   // TODO: Rest of the langauges
```
Comment (Albert221) [inline lib/models/language.dart]: It's no more needed here.

**Review by Albert221 — CHANGES_REQUESTED** [review]
Some minor stuff to change.

File: lib/models/language.dart (hunk 2 above)
Comment (MatejMecka) [inline lib/models/language.dart]:
```suggestion
  static const frenchCanadian = Language('fr-CA', 'French (Canada)');
```

File: lib/models/language.dart [hunk 8]
```diff
@@ -41,8 +41,35 @@ abstract class GooglePlayLanguages {
...
+  static const malay = Language('ms', 'Malay');
+  static const norwegian = Language('no', 'Norwegian');
+  static const portugueseBr = Language('pt-BR', 'Portuguese (Brazil)');
+  static const portuguesePr = Language('pt-PT', 'Portuguese (Portugal) ');
+  static const romanian = Language('ro', 'Romanian');
+  static const russian = Language('ru', 'Russian');
+  static const serbian = Language('sr', 'Serbian');
+  static const slovak = Language('sk', 'Slovak');
+  static const slovenian = Language('sl', 'Slovenian'); 
+  static const spanishLa = Language('es-419', 'Spanish (Latin America)');
```
Comment (Albert221) [inline lib/models/language.dart]: Please expand `La` to `LatinAmerica`

File: lib/models/language.dart [hunk 9]
```diff
@@ -41,8 +41,35 @@ abstract class GooglePlayLanguages {
...
+  static const icelandic = Language('is', 'Icelandic');
+  static const indonesian = Language('id', 'Indonesian');
+  static const italian = Language('it', 'Italian');
+  static const korean = Language('ko', 'Korean');
+  static const latvian = Language('lv', 'Latvian');
+  static const lithuanian = Language('lt', 'Lithuanian');
+  static const malay = Language('ms', 'Malay');
+  static const norwegian = Language('no', 'Norwegian');
+  static const portugueseBr = Language('pt-BR', 'Portuguese (Brazil)');
+  static const portuguesePr = Language('pt-PT', 'Portuguese (Portugal) ');
```
Comment (Albert221) [inline lib/models/language.dart]: The trailing space is still here. Also, expand `Pr` to `Portugal` please

File: lib/models/language.dart [hunk 10]
```diff
@@ -41,8 +41,35 @@ abstract class GooglePlayLanguages {
...
+  static const hindi = Language('hi', 'Hindi');
+  static const icelandic = Language('is', 'Icelandic');
+  static const indonesian = Language('id', 'Indonesian');
+  static const italian = Language('it', 'Italian');
+  static const korean = Language('ko', 'Korean');
+  static const latvian = Language('lv', 'Latvian');
+  static const lithuanian = Language('lt', 'Lithuanian');
+  static const malay = Language('ms', 'Malay');
+  static const norwegian = Language('no', 'Norwegian');
+  static const portugueseBr = Language('pt-BR', 'Portuguese (Brazil)');
```
Comment (Albert221) [inline lib/models/language.dart]: Expand `Br` to `Brazil`

**Review by Albert221 — CHANGES_REQUESTED** [review]

File: lib/models/language.dart [hunk 11]
```diff
@@ -41,8 +41,35 @@ abstract class GooglePlayLanguages {
   static const filipino = Language('fil', 'Filipino');
   static const finnish = Language('fi-FI', 'Finnish');
   static const french = Language('fr-FR', 'French');
-  // TODO: Rest of the langauges
-
+  static const frenchCanadian = Language('fr-CA', 'French (Canada)');
```
Comment (Albert221) [inline lib/models/language.dart]: Canadian => Canada

**Review by Albert221 — CHANGES_REQUESTED** [review]

**Review by Albert221 — APPROVED** [review]

**Pull request merged**